src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static String getRandomColor() {
+        Random random = new Random();
+        int value = random.nextInt(Colors.values().length);
+        return Colors.values()[value].toString();
```

Comment:
```suggestion
        return Colors.values()[value].name();
```
<img width="1078" alt="image" src="https://user-images.githubusercontent.com/81781479/203365468-16598692-a168-48f0-b620-199ac0a6e9ba.png">